plugins/SolidView/SolidView.py

```diff
@@ -67,20 +67,27 @@ def __init__(self):
         self._xray_checking_update_time = 30.0 # seconds
         self._xray_warning_cooldown = 60 * 10 # reshow Model error message every 10 minutes
         self._xray_warning_message = Message(
-            catalog.i18nc("@info:status", "Your model is not manifold. The highlighted areas indicate either missing or extraneous surfaces."),
+            catalog.i18nc("@info:status", "The highlighted areas indicate either missing or extraneous surfaces. Fix your model and import it again into Cura."),
```

Comment:
For consistency, I think Cura uses the verb "open" instead of "import" for 3d models.

Reply:
Fixed!